aws/lambdas/nagware.tf

```diff
@@ -39,7 +39,7 @@ resource "aws_lambda_function" "nagware" {
       DB_ARN                    = var.rds_cluster_arn
       DB_SECRET                 = var.database_secret_arn
       DB_NAME                   = var.rds_db_name
-      NOTIFY_API_KEY            = var.notify_api_key_secret_arn
+      NOTIFY_API_KEY            = var.notify_api_key_secret_value
```

Comment:
One thing to watch for with this is that this value can leak more easily now.  We had cases in another projects where AWS cli commands that managed the Lambda function were leaking the `environment` vars as part of their response.

If you have any spots where you're doing things with the `aws lambda` cli commands, you'll need to pipe standard out and error to `/dev/null`:
https://github.com/cds-snc/scan-files/blob/cc153d6cad51aaac3ce77a89f5b2aacf93b97495/.github/workflows/build_and_push_staging.yml#L77-L79

Reply:
That is correct! We are only using the key when we initialize the `NotifyClient` for now so there should not be any risk (it was working the same way before the infra update anyway)